lib/omnibus/project.rb

```diff
@@ -928,6 +967,11 @@ def build_me
       # Health check
       HealthCheck.run!(self)
 
+      # Remove any package this project extends, after the health check ran
+      extended_packages.each do |package|
+        packager.remove(package)
```

Comment:
same here for a list of packages, also depending on the type of files installed by this package (config files or such), you might want to consider using `purge` instead of `remove` ?